tmt/cli.py

```diff
@@ -429,8 +433,18 @@ def create(context, name, template, force, **kwargs):
 @verbose_debug_quiet
 @force_dry
 def import_(
-        context, paths, makefile, nitrate, purpose, disabled, manual, plan,
-        case, with_script, **kwargs):
+        context,
+        paths,
+        makefile,
+        type,
+        nitrate,
+        purpose,
+        disabled,
+        manual,
+        plan,
+        case,
+        with_script,
+        **kwargs):
```

Comment:
No need to keep one parameter per line, `pre-commit` just ensures the `79` chars limit is not exceeded. Also it's recommended to prevent collision with reserved python words using a trailing underscore, so let's call the new parameter `type_` then.